src/System.IO.Compression/src/System/IO/Compression/DeflateZLib/Deflater.cs

```diff
@@ -134,9 +134,12 @@ internal int GetDeflateOutput(byte[] outputBuffer)
 
         private unsafe ZErrorCode ReadDeflateOutput(byte[] outputBuffer, ZFlushCode flushCode, out int bytesRead)
         {
+            Debug.Assert(null != outputBuffer, "Can't pass in a null output buffer!");
```

Comment:
> I believe he's trying to remain consistent with the rest of them, and avoid changing those others in this PR as they're not related to its primary focus.

Yes, this is what I meant. In the end, I decided to change the new asserts I added here to a single `Debug.Assert(outputBuffer?.Length > 0);`, similar to the asserts I've added elsewhere in this PR, but didn't go through and modify the existing asserts throughout this file. That can be done separately, if desired.